main.py

```diff
@@ -154,14 +157,24 @@ def get_Announcements():
 def get_Pair_and_DateTime(ARTICLE_CODE):
     
     new_Coin = requests.get(ARTICLE+ARTICLE_CODE).json()['data']['seoDesc']
-    datetime = dparser.parse(new_Coin, fuzzy=True, ignoretz=True)
-    
-    raw_pairs = re.findall(regex, new_Coin)
-    pairs = []
+    try:
+        datetime = dparser.parse(new_Coin, fuzzy=True, ignoretz=True)
     
-    for pair in raw_pairs:
-        pairs.append(pair.replace('/', ''))
-    return [datetime, pairs]
+        raw_pairs = re.findall(regex, new_Coin)
+        pairs = []
+
+        for pair in raw_pairs:
+            present= False
+            for j in existing_assets:
+                if j in pair:
+                    present = True
+                    break
+            if present == False:
+                pairs.append(pair.replace('/', ''))
+        return [datetime, pairs]
+    except Exception as e:
+        print(e)
```

Comment:
better to use the defined message function so headless instances doesn't need to check console